js/options/options.js

```diff
@@ -440,20 +440,33 @@ let Options = (function(){
             let setting = node.dataset.setting;
             let value = SyncedStorage.get(setting);
 
-            if (node.type && node.type === "checkbox") {
+            // If the node is a dropdown, check if the option has a sub option that gets triggered by it.
+            if (node.tagName === "SELECT") {
+                if (value) {
+                    node.value = value;
+                }
+
+                let parentOption = node.closest(".parent_option");
+                if (parentOption && parentOption.dataset.dropdownTrigger) {
+                    let disabled = value !== parentOption.dataset.dropdownTrigger;
+                    for (let nextSibling = parentOption.nextElementSibling; nextSibling && nextSibling.classList.contains("sub_option"); nextSibling = nextSibling.nextElementSibling) {
+                        nextSibling.classList.toggle("disabled", disabled);
+                    }
+                }
+            } else if (node.type && node.type === "checkbox") {
                 node.checked = value;
 
                 let parentOption = node.closest(".parent_option");
-                if (parentOption) {
+                if (parentOption && !parentOption.dataset.dropdownTrigger) {
                     if (node.id === "stores_all") value = !value;
-                    for (let nextSibling = parentOption.nextElementSibling; nextSibling.classList.contains("sub_option"); nextSibling = nextSibling.nextElementSibling) {
+                    for (let nextSibling = parentOption.nextElementSibling; nextSibling && nextSibling.classList.contains("sub_option"); nextSibling = nextSibling.nextElementSibling) {
```

Comment:
Had to add this because it results in an error when the sub option is the last element child.